test cases/common/139 simd/meson.build

```diff
@@ -0,0 +1,31 @@
+project('simd', 'c')
+
+simd = import('simd')
+
+cc = meson.get_compiler('c')
+
+cdata = configuration_data()
+
+simdlibs = simd.check('mysimds',
```

Comment:
No because we don't support per-file compiler arguments and the calling code _must_ be compiled without the argument. And further just returning the "best" list is not good enough, we need to support the case where people want to ship code that has many different SIMD implementations of which the best one is selected at runtime.